### PR TITLE
support multi containers

### PR DESCRIPTION
**Pull request by Ox0400**
support multi containers like this
```

frontend http_in_80_1
    bind *:80
    mode http

    acl is_rule_localhost_80_1_1 hdr(host) -i localhost
    acl is_rule_localhost_80_1_2 hdr(host) -i localhost:80
    use_backend srv_localhost_80_1 if is_rule_localhost_80_1_1 OR is_rule_localhost_80_1_2

backend srv_localhost_80_1
    balance roundrobin
    mode http
    option forwardfor
    http-request set-header X-Forwarded-Port %[dst_port]
    http-request add-header X-Forwarded-Proto https if { ssl_fc }
    server a0b8ccb2d40f:80 a0b8ccb2d40f:80 check weight 1
    server f718c6f0f6c3:80 f718c6f0f6c3:80 check weight 1



```

### CODE REVIEW

File: templates/haproxy.cfg.j2 [hunk 1]
```diff
@@ -62,6 +62,8 @@ backend srv_{{ host }}
     option tcp-check
     tcp-check connect{{ " ssl" if o["health-check"] == "ssl" }}
         {% endif %}
-    server srv {{ o["hosts"][k] }} check weight 1{{ " verify none" if o["health-check"] == "ssl" }}
+        {% for c in o["hosts"][k] %}
+    server srv {{ c }} check weight 1{{ " verify none" if o["health-check"] == "ssl" }}
```
Comment (Ox0400) [inline templates/haproxy.cfg.j2:66]: srv need uniq

**Comment by byjg** [comment]
Thank you for your PR. I am analyzing it, meanwhile, could you back merge from master? I added today the GitHub actions test and build.

File: templates/haproxy.cfg.j2 [hunk 2]
```diff
@@ -62,6 +62,8 @@ backend srv_{{ host }}
     option tcp-check
     tcp-check connect{{ " ssl" if o["health-check"] == "ssl" }}
         {% endif %}
-    server srv {{ o["hosts"][k] }} check weight 1{{ " verify none" if o["health-check"] == "ssl" }}
+        {% for c in o["hosts"][k] %}
+    server {{ c }} {{ c }} check weight 1{{ " verify none" if o["health-check"] == "ssl" }}
```
Comment (byjg) [inline templates/haproxy.cfg.j2:66]: Could you replace `server {{ c }} {{ c }} ...` by `server srv{{ incr }} {{ c }} ...`

result output e.g.:

```
server srv1 test_agent:9001 check weight 1 verify none
```

https://cbonte.github.io/haproxy-dconv/2.4/configuration.html#5.2

Comment (Ox0400) [reply templates/haproxy.cfg.j2]: updated.

```
    server srv-0 f6d8d45b7411:80 check weight 1
    server srv-1 59b213cb8592:80 check weight 1

```

**Comment by byjg** [comment]
Thank you. I am merging and soon creating a release tag

**Comment by byjg** [comment]
Docker image built. 

The tags `latest` and `3.0.2` now contain mult-containers. Thanks.

**Pull request merged**